package.json

```diff
@@ -56,6 +56,7 @@
     "eslint-plugin-simple-import-sort": "^4.0.0",
     "esm": "^3.2.25",
     "fs-extra": "^8.1.0",
+    "git-describe": "^4.0.4",
```

Comment:
Do we need this dependency? Or can we use git describe directly?

Reply:
It's an exceptionally small module and it's only in `devDependencies`, so I'm happy to save work where I can.